cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/resultset/PagesInIntervalIteratorProvider.java

```diff
@@ -72,7 +75,10 @@ private Queue<String> getPagesInInterval(BookId bookId, BookCache bookCache, Int
         return bookCache.loadPageInfo(bookId, false)
                 .stream()
                 .filter(page -> checkInterval(page, start, end))
-                .map(page -> page.getId().getName())
+                .map(page -> {
+                    logger.info("page: {} start: {} - end: {}", page.getId(), page.getStarted(), page.getEnded());
```

Comment:
Please reduce logging level or remove the login at all

Reply:
it was just for testing, it was planed to be removed after issue is fixed